components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/api/impl/BusinessRulesApiServiceImpl.java

```diff
@@ -176,7 +175,7 @@ public Response loadBusinessRule(String businessRuleInstanceID
     @Override
     public Response redeployBusinessRule(String businessRuleInstanceID
     ) throws NotFoundException {
-        TemplateManagerService templateManagerService = new TemplateManagerService();
+        TemplateManagerService templateManagerService = TemplateManagerInstance.getInstance();
         int status;
```

Comment:
make this a one line

Reply:
Fixed by 4589e7a6cad0014f57d6fc9e8a46734d291d733f